Diagonal Traverse.py

```diff
@@ -0,0 +1,20 @@
+# Time Complexity: O(n)
+# Space Complexity: O(n)
+# This code was passsed on LC:  no
+# Problems: Not able to solve
+
```

Comment:
The logic seems to be wrong, try out as taught in class and let me know if you still find it difficult.